src/News Articles/NewsArticles.tsx

```diff
@@ -45,7 +38,7 @@ export const NewsArticles: FunctionComponent = () => {
     const [articles, setArticles] = useState<Article[]>([]);
 
     useEffect(() => {
-        fetchNewsArticles()
+        void fetchNewsArticles()
```

Comment:
ooh weird - is this for a linting rule?
if so we should turn it off! 😂

Reply:
Was to satisfy the no-floating-promises rule!